koku/masu/external/downloader/aws/aws_report_downloader.py

```diff
@@ -72,7 +80,7 @@ def get_initial_dataframe_with_date(
                 s3_csv_path, start_date, end_date, manifest_id, context, tracing_id
             )
             ReportManifestDBAccessor().set_manifest_daily_start_date(manifest_id, process_date)
-    return data_frame, time_interval, process_date
+    return use_cols, time_interval, process_date
```

Comment:
This is a nitpick and not critical, but this function no longer returns a dataframe, but is still named: `get_initial_dataframe_with_date`.